databricks/koalas/series.py

```diff
@@ -1354,6 +1354,96 @@ def sort_index(self, axis: int = 0, level: int = None, ascending: bool = True,
         else:
             return ks_
 
+    def add_prefix(self, prefix):
+        """
+        Prefix labels with string `prefix`.
+
+        For Series, the row labels are prefixed.
+
+        Parameters
+        ----------
+        prefix : str
+           The string to add before each label.
+
+        Returns
+        -------
+        Series
+           New Series with updated labels.
+
+        See Also
+        --------
+        Series.add_suffix: Suffix column labels with string `suffix`.
+        DataFrame.add_suffix: Suffix column labels with string `suffix`.
+
+        Examples
+        --------
+        >>> s = ks.Series([1, 2, 3, 4])
+        >>> s
+        0    1
+        1    2
+        2    3
+        3    4
+        Name: 0, dtype: int64
+
+        >>> s.add_prefix('item_')
+        item_0    1
+        item_1    2
+        item_2    3
+        item_3    4
+        Name: 0, dtype: int64
+        """
+        assert isinstance(prefix, str)
+        kdf = self._kdf
```

Comment:
This logic overwrites the original `kdf`.
We should start with `kdf = self.to_dataframe()`.